### PR TITLE
additions to admin - look at individual library info (members & keeps)

### PR DESCRIPTION
**Pull request by ahsu1230**
Adding to admin page...

when visiting libraries tab, you can visit each individual library
when looking at each individual library, you can also view all keeps in that library

@andrewconner @stkem @eishay @2is10

### CODE REVIEW

File: kifi-backend/modules/common/app/assets/stylesheets/admin_library.less [hunk 1]
```diff
@@ -0,0 +1,4 @@
+table.library-summary {
+  margin-top: 10px;
+  width: auto;
+}
```
Comment (stkem) [inline kifi-backend/modules/common/app/assets/stylesheets/admin_library.less]: newline please :-)

Comment (ahsu1230) [inline kifi-backend/modules/common/app/assets/stylesheets/admin_library.less]: lol just caught that as well

**Comment by stkem** [comment]
can't claim to be able to parse play templates fully just by looking at them, but the rest lgtm

**Comment by eishay** [comment]
does the admin page allow us to see libraries that are not browsable? I.e. private keeps?

**Comment by ahsu1230** [comment]
yes, when you look at keeps in a library, you can see which keeps are private vs. public (though they should all be one or the other anyway). So, you can see keeps in a secret library

**Comment by ahsu1230** [comment]
@stkem sorry, for some reason the files of play templates look different on github.
Like tabs just got messed up or something

**Comment by eishay** [comment]
can we hide from the admin keeps that people kept privately? I don't want to accidentally see those unless I must

**Comment by andrewconner** [comment]
Check `showPrivates` for Keeps, @ahsu1230.

**Comment by stkem** [comment]
+1 on that

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminLibraryController.scala [hunk 2]
```diff
@@ -47,6 +50,40 @@ class AdminLibraryController @Inject() (
...
+
+  def libraryKeepsView(libraryId: Id[Library], showPrivates: Boolean = false) = AdminHtmlAction.authenticated { implicit request =>
+    if (showPrivates) {
+      log.warn(s"${request.user.firstName} ${request.user.firstName} (${request.userId}) is viewing private library $libraryId")
+    }
+
+    val (library, owner, totalKeepCount, keepInfos) = db.readOnlyReplica { implicit session =>
+      val lib = libraryRepo.get(libraryId)
+      val owner = userRepo.get(lib.ownerId)
+      val keeps = keepRepo.getByLibrary(libraryId).filter(b => showPrivates || !(b.isPrivate || lib.visibility == LibraryVisibility.SECRET))
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminLibraryController.scala:76]: here's the privacy checker @eishay @stkem @andrewconner 
along with a log warning a few lines up

Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminLibraryController.scala:76]: :+1:

**Pull request merged**